contracts/gateway/tests/contract.rs

```diff
@@ -314,8 +313,8 @@ fn reject_reroute_outgoing_message_with_different_contents() {
     );
     assert!(response.is_err_and(|err| err_contains!(
         err.report,
-        state::Error,
-        state::Error::MessageMismatch { .. }
+        Error,
+        Error::RouteOutgoingMessages
```

Comment:
why did this change?

Reply:
`state::Error` isn't exposed anymore, see below. Thoughts?
https://github.com/axelarnetwork/axelar-amplifier/pull/582#discussion_r1714140803